Tools/Create-Azure-Sentinel-Solution/createSolution.ps1

```diff
@@ -358,6 +358,22 @@ foreach ($inputFile in $(Get-ChildItem $path)) {
                                         minLength = 1;
                                         metadata  = [PSCustomObject] @{ description = "Password to connect to $solutionName API"; }
                                     })
+                            }
```

Comment:
please ensure proper formatting is maintained.

Reply:
Done.